controllers/events.js

```diff
@@ -0,0 +1,63 @@
+const eventRouter = require('express').Router()
+const verifyService = require('../services/verifyService')
+const eventService = require('../services/eventService')
+
+// TODO: verify if the scout is actually logged in ..?
+
+
+// Get a list of scouts events
+eventRouter.get('', async (req, res) => {
+  const scout = req.session.scout
```

Comment:
Should this use `parseInt(req.session.scout)`? At least weird calls like /events/ghfdgr should be handled somehow. Same goes for /activities.